packages/analytics-js-integrations/src/integrations/Optimizely/browser.js

```diff
@@ -153,20 +162,34 @@ class Optimizely {
 
     const registerCurrentlyActiveCampaigns = () => {
       window.optimizely = window.optimizely || [];
-      const state = window?.optimizely?.get('state');
-      if (state) {
-        const referrer = checkReferrer();
-        const activeCampaigns = state.getCampaignStates({
-          isActive: true,
-        });
-        Object.keys(activeCampaigns).forEach(id => {
-          if (referrer) {
-            newActiveCampaign(id, referrer);
-          } else {
-            newActiveCampaign(id);
-          }
-        });
-      } else {
+      try {
+        const state = window?.optimizely?.get('state');
```

Comment:
```suggestion
let state;
      try {
        state = window?.optimizely?.get('state');
        } catch(){
        state=undefined
        }
```
Do it this way. No need to repeat the below code
```
window.optimizely.push({
            type: 'addListener',
            filter: {
              type: 'lifecycle',
              name: 'initialized',
            },
            handler() {
              checkReferrer();
            },
          });
        }
```